src/Aspire.Hosting/Dcp/ApplicationExecutor.cs

```diff
@@ -494,13 +487,16 @@ private void StartLogStream<T>(T resource) where T : CustomResource
                     {
                         foreach (var (content, isError) in batch)
                         {
-                            if (!ResourceLoggerService.TryParseContentLineDate(content, out var dateTimeUtc))
+                            DateTime? timestamp = null;
+                            var resolvedContent = content;
+
+                            if (TimestampParser.TryParseConsoleTimestamp(resolvedContent, out var result))
```

Comment:
Why do we parse the timestamp out here, just to turn around and put them back together in the `ResourceLoggerService`? Why not just leave the timestamp in the content, and pass it all the way up to the dashboard. And only need to parse it out once in the UI?

Reply:
I did this so the content in the host LogEntry was consistent with the client LogEntry.

Reply:
Why do we need them to be consistent? The host can just deal with the logs as lines of content.

it’s only the UI that wants them to be sorted by date time, so why would we force the host to parse the time out?

Reply:
They don't need to be consistent; I just think it's simpler to reason about everything when it's consistent and it allows more code to be shared between the host and dashboard.

In the host, either the `ILogger` logs need to add a timestamp to log line content or DCP logs need to remove a timestamp from log line content. Either way, it's slightly weird.

I chose to make the DCP logs remove the time stamp over the alternative for the reasons above. The downside there are some extra string allocations, the upside is storing the backlog uses less memory. Date a long `DateTime` is smaller than vs 15ish UTF16 chars in the log message.